freqtrade/rpc/api_server/api_schemas.py

```diff
@@ -347,3 +347,8 @@ class BacktestResponse(BaseModel):
     trade_count: Optional[float]
     # TODO: Properly type backtestresult...
     backtest_result: Optional[Dict[str, Any]]
+
+
+class SysInfo(BaseModel):
+    cpu_pct: float
```

Comment:
Your type seems wrong.
While i've added a test now - cpu_pct is not returning a float - so the test fails.
fixing the type here should also fix the test.